Specs/Core/OrientedBoundingBoxSpec.js

```diff
@@ -0,0 +1,417 @@
+        pl = planeNormXform(-1.0, -1.0, +1.0, -SQRT3_4 + 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.INTERSECTING); }
+        pl = planeNormXform(-1.0, -1.0, -1.0, -SQRT3_4 + 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.INTERSECTING); }
+
+        pl = planeNormXform(+1.0, +1.0, +1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(+1.0, +1.0, -1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(+1.0, -1.0, +1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(+1.0, -1.0, -1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(-1.0, +1.0, +1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(-1.0, +1.0, -1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(-1.0, -1.0, +1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+        pl = planeNormXform(-1.0, -1.0, -1.0, -SQRT3_4 - 0.00001); if (pl) { expect(box.planeIntersect(pl)).toEqual(Intersect.OUTSIDE); }
+
+    };
+
+    it('planeIntersect works with untransformed box', function() {
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.IDENTITY);
+    });
+
+    it('planeIntersect works with off-center box', function() {
+        planeIntersectTestCornersEdgesFaces(new Cartesian3(1.0, 0.0, 0.0), Matrix3.IDENTITY);
+        planeIntersectTestCornersEdgesFaces(new Cartesian3(0.7, -1.8, 12.0), Matrix3.IDENTITY);
+    });
+
+    it('planeIntersect works with rotated box', function() {
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO,
+                Matrix3.fromQuaternion(Quaternion.fromAxisAngle(new Cartesian3(0.5, 1.5, -1.2), 1.2), new Matrix3()));
+    });
+
+    it('planeIntersect works with scaled box', function() {
+        var m = new Matrix3();
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.fromScale(new Cartesian3(1.5, 0.4, 20.6), m));
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.fromScale(new Cartesian3(0.0, 0.4, 20.6), m));
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.fromScale(new Cartesian3(1.5, 0.0, 20.6), m));
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.fromScale(new Cartesian3(1.5, 0.4, 0.0), m));
+        planeIntersectTestCornersEdgesFaces(Cartesian3.ZERO, Matrix3.fromScale(new Cartesian3(0.0, 0.0, 0.0), m));
+    });
+
+    //it('planeIntersect works with negatively scaled box', function() {
```

Comment:
I didn't completely delete this because I wasn't sure if it was worth testing. However, I'm pretty sure it isn't.